src/state/form/actions.js

```diff
@@ -0,0 +1,28 @@
+
+import { gotoRoute } from 'frontend-common-components';
+import { login } from 'api/login';
+
+import { SET_LOGIN_ERROR_MESSAGE } from './types';
+
+
+// eslint-disable-next-line
+export const setLoginErrorMessage = (message) => ({
+  type: SET_LOGIN_ERROR_MESSAGE,
+  payload: {
+    message,
+  },
+});
+
+
+// thunks
+
+export const performLogin = (username, password) => (dispatch) => {
+  if (username && password) {
+    dispatch(setLoginErrorMessage(''));
+    login(username, password).then(() => {
+      gotoRoute('dashboard');
+    });
+  } else {
+    dispatch(setLoginErrorMessage('ERROR: USERNAME OR PASSWORD IS INVALID'));
```

Comment:
Please use `formattedText` function and create a message in locales/en and locales/it